README.md

```diff
@@ -52,78 +75,91 @@ npm start
 
 ## Development
 
-#### Requirements
-
-- Node.js v6.x.x
-- npm v3.x.x
+### Requirements
+- [Node.js v4.x.x](http://nodejs.org/)
+- [npm](https://npmjs.com)
 - g++, gcc, make (for building native modules)
 - python 2 (for building, some native modules need it, node-fibers perhaps?)
 
-#### Get the source code
+The following npm modules must be installed globally: [grunt-cli](https://www.npmjs.com/package/grunt-cli), [mocha](https://www.npmjs.com/package/mocha), [electron-prebuilt](https://www.npmjs.com/package/electron-prebuilt). These can be installed by running:
+
+```sh
+npm i -g grunt-cli mocha electron-prebuilt
```

Comment:
Ran into this before; can't check it now, as my train wifi is too poor for `npm i`. Will come back to this.